content/blog/2025/04/2025-04-28-election-announcement.md

```diff
@@ -0,0 +1,97 @@
+* We do not require people to use their “legal name.”
+* Candidates who are nominated by an organizational member will have their affiliation listed.
+* We expect candidates to provide a short bio and a short statement, collectively covering qualifications and priorities. Candidates may also provide a link to an extended statement or platform on their personal blog or website.
+* You are encouraged to make yourself available for conversation with members of your constituency group.
+* The period for campaigning runs from April 27th to May 17th. Please note our [community Code of Conduct](https://matrix.org/legal/code-of-conduct/) while campaigning.
+
+## What to expect if you win
+
+Congratulations! All of the winners will be announced on June 16th, but since the elections conclude on June 13th, you may get an email with the results of the election you participated in before June 16th.
+
+All elected representatives will be added to a private mailing list and Matrix room so that they can introduce themselves and communicate in between meetings. I will be reaching out to every elected representative to meet one-on-one, get acquainted, and answer questions before we convene for their first Governing Board meeting.
+
+The Governing Board meets as a full board twice a year, for at least 90 minutes each time, and meetings are facilitated by the duly elected Chair or Vice Chair. Expect one meeting before the end of the year, and another before the next election.
+
+You are likely to receive a packet of materials before each meeting which you will be expected to have reviewed. Most meetings will be held online, though we may convene meetings in-person, such as at The Matrix Conference. We will make every effort to make in-person meetings accessible to remote participants, and no official business will ever happen without appropriate notice, quorum, and minutes.
+
+While the Governing Board is an advisory board, members are encouraged to play a role in helping to carry out the activities that support the staff in delivering on the Foundation’s collective remit, such as coalescing priorities to share with the Spec Core Team and connecting with contributors and funders to implement proposals.
+
+Most of the work of the Governing Board happens in its Committees and their Working Groups, who meet in between Governing Board meetings and present their assessments and recommendations to the Governing Board. 
+
+## How to nominate
+
+We will update this blog post with a link to a nomination form and a template document on Friday, May 2nd. We’ll be reaching out to everyone who is eligible and whose contact information we have to let them know. We’ll also be announcing it on our social channels.
+
+All nominations should be submitted through the form, though anyone who has technical difficulties can fill out the template document and email it to [elections@foundation.matrix.org](mailto:elections@foundation.matrix.org).
```

Comment:
We could also link to the GB's office matrix room as the central place for discussion on matrix? alternatively, the fdn's office.

Reply:
Oh yes, love the idea of linking to the GB Office